### PR TITLE
CI: use 16 cores instead of 8 cores

### PR DESCRIPTION
**Pull request by ma2bd**
## Motivation

Faster CI

## Proposal

Use Ubuntu 16 cores instead of 8 cores

## Test Plan

CI

Comparing to #1067:
- Scylladb goes from 8min to 5min
- DynamodDb goes from 9min to 8min :-|
- Rust goes from 37min to 28min

### CODE REVIEW

**Comment by ma2bd** [comment]
Please ignore the first commit (I will remove it before landing)

**Review by andresilva91 — APPROVED** [review]
LGTM! Just wait for CI to be green and ship it!
[movefast](https://app.graphite.dev/macros/linera-io)

**Pull request merged**
